data/account/participationRegistry.go

```diff
@@ -344,6 +345,7 @@ const (
 	insertKeysetQuery         = `INSERT INTO Keysets (participationID, account, firstValidRound, lastValidRound, keyDilution, vrf, stateProof) VALUES (?, ?, ?, ?, ?, ?, ?)`
 	insertRollingQuery        = `INSERT INTO Rolling (pk, voting) VALUES (?, ?)`
 	appendStateProofKeysQuery = `INSERT INTO StateProofKeys (pk, round, key) VALUES(?, ?, ?)`
+	deleteStateProofKeysQuery = `DELETE FROM StateProofKeys WHERE pk=? AND round<=?`
```

Comment:
We can, but it would imply doing a table migration which I'm not sure worth it.
anyways, in the long run, we would like to get rid of the `persistentMerkleSignatureScheme` since we are using the participation registry